templates/deployment.yaml

```
@@ -32,7 +37,7 @@ spec:
   selector:
     matchLabels: {{- include "common.selectorLabels" ( dict "root" $ "service" $serviceDefinition "serviceName" $serviceName ) | nindent 6 }}
   {{- end }}
-  {{- if eq ( default $serviceDefinition.type "Deployment" ) "StatefulSet" }}
+  {{- if eq ( $serviceDefinition.type | default "Deployment" ) "StatefulSet" }}
```

This mix it difficult to read => 
```suggestion
  {{- if eq ( default "Deployment" $serviceDefinition.type ) "StatefulSet" }}
```